hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3MultipartUploadAbortRequest.java

```diff
@@ -120,6 +124,21 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
         multipartKeyInfo = omMetadataManager
             .getMultipartInfoTable().get(multipartKey);
 
+        // Set updateID to current transactionLogIndex for all parts
+        TreeMap<Integer, PartKeyInfo> partKeyInfoMap = multipartKeyInfo
+            .getPartKeyInfoMap();
+        for (Map.Entry<Integer, PartKeyInfo> partKeyInfoEntry :
+            partKeyInfoMap.entrySet()) {
+          PartKeyInfo partKeyInfo = partKeyInfoEntry.getValue();
+          OmKeyInfo currentKeyPartInfo = OmKeyInfo.getFromProtobuf(
```

Comment:
Not understood the reason of setting updateID here, as below we are setting Optional.absent(), so it  means entry is deleted from DB.

Reply:
Yes, but this entry is added to DeletedTable. So I thought we should keep a record of the transaction which deleted the Key.